packages/vue-client/src/components/RuleDescriptions/FractionalRules.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <h2>Base Setup</h2>
+
+  <p>
+    Fractional is a multiplayer variant for any even number of participants.
+    Players are grouped into two competing teams:
+  </p>
+  <ul>
+    <li><strong>Team Black</strong></li>
+    <li><strong>Team White</strong></li>
+  </ul>
```

Comment:
That description is too specific to use it for *rules*, it might be better to use it as an example. The variant also allows for other colors to be primary colors. For example there could be three teams (red, green, blue) of three players each (cyan, magenta, yellow).

Here's a suggestion for a different phrasing, but feel free to change it:

> Fractional Go is a multiplayer variant with multicolored stones. Usually players are grouped into two competing teams:

Reply:
Should I leave out the teams aspect? Perhaps it's sufficient to describe the objective with something like

_Players place stones of two colours, one being the **Primary Colour** and one being the **Secondary Colour**. At the end of a game, players receive points equal to the area controlled by their primary colour._

Reply:
I made the rules description more generic, to reflect the flexibility of the variant.

I like the idea of an additional explanation section, but for now I'm content with adding just the rules.